cypress/integration/Carousel.feature

```diff
@@ -0,0 +1,102 @@
+Feature: Carousel component
+  I want to change Carousel properties
+
+  Background: Open Carousel component page
+    Given I open "Carousel" component page
+
+  @positive
+  Scenario: Initial slide title is Slide One
+    #When I open "Carousel" component page
```

Comment:
Commented line of code should be deleted :)

Reply:
fixed